src/ICS20Transfer.sol

```diff
@@ -233,7 +223,7 @@ contract ICS20Transfer is IIBCApp, IICS20Transfer, IICS20Errors, Ownable, Reentr
         string calldata destChannel,
         ICS20Lib.FungibleTokenPacketData memory packetData
     )
-        private
+        public
```

Comment:
Note this, I changed the visibility here to have access to this function to properly test the logic of the foreignDenoms test without relying on the events. However it may not be the ideal solutions.